cypress/e2e/functional/researcher_reports.spec.js

```diff
@@ -0,0 +1,80 @@
+import * as c from '../../support/constants.js'
+import {ResearcherReportsPageElements as researcherReport} from "../../support/elements/researcher_reports_page_elements.js";
+
+context("Researcher reports UI", () => {
+
+  before(function() {
+    cy.visit(c.LEARN_PORTAL_BASE_URL);
+    cy.login(c.ADMIN_USERNAME, c.ADMIN_PASSWORD);
+    cy.visit(c.LEARN_PORTAL_BASE_URL + '/admin');
+    cy.contains("Administration and Reports");
+  });
+
+  function logout() {
+    cy.logout();
+    cy.clearAllCookies();
+  }
+
+  it("Researcher Reports ", () => {
+    cy.log("launch learner reports");
+    cy.contains("Learner Reports").invoke("removeAttr", "target").click();
+    cy.loginPortal(c.ADMIN_USERNAME, c.ADMIN_PASSWORD);
+  
+    cy.log("Verify research reports fields and label display");
+    researcherReport.verifyUserName();
```

Comment:
As I already commented in researcher_reports_page_elements.js, the username "joe user" should be given here, not in that lower level file.

Reply:
Done